modules/fts/pages/fts-index-analyzers.adoc

```diff
@@ -12,20 +12,23 @@ An analyzer consists of modules, each of which performs a particular, sequenced
 _Analyzers_ pre-process input-text submitted for Full Text Search; typically, by removing characters that might prohibit certain match-options.
 
 The analysis is performed on document-contents when indexes are created and is also performed on the input-text submitted for a search.
-The benefit of analysis is often referred to as _language awareness_.
+This enables _language awareness_ for FTS..
 
-For example, if the input-text for a search is `enjoyed staying here`, and the document-content contains the phrase `many enjoyable activities`, the dictionary-based words do not permit a match.
-However, by using an analyzer that (by means of its inner _Token Filter_ component) _stems_ words, the input-text yields the tokens `enjoy`, `stay`, and `here`; while the document-content yields the tokens `enjoy` and `activ`.
-By means of the common token `enjoy`, this permits a match between `enjoyed` and `enjoyable`.
+For example, if the input-text for a search is `liked staying here`, and the document-content contains the phrase `many things to like`, the dictionary-based words do not permit a match.
+However, by using an analyzer (_en_ for this example) that _tokenizes_ and _stems_ words by means of its inner _Tokenizer_ and _Token Filter_ components, the input-text yields the tokens `like` and `stai`; while the document-content yields the tokens `mani`, `thing` and `like`.
+By means of the common token `like`, this permits a match between `like` and `liked`.
 
 Since different analyzers pre-process text in different ways, effective Full Text Search depends on the right choice of the analyzer, for the type of matches that are desired.
 
 Couchbase Full Text Search provides a number of pre-constructed analyzers that can be used with Full Text Indexes.
-Additionally, analyzers can be custom-created by means of the Couchbase Web Console.
+Additionally, analyzers can be custom-created on the Couchbase Web Console or using the http://bleveanalysis.couchbase.com/analysis[Text Analysis Wizard].
+
 The remainder of this page explains the architecture of analyzers and describes the modular components that Couchbase Full Text Search makes available for custom-creation.
 It also lists the pre-constructed analyzers that are available and describes the modules that they contain.
 
-For examples of both selecting and custom-creating analyzers by means of the Couchbase Web Console.
+Also, here's a relevant article on https://blog.couchbase.com/full-text_search_text_analysis/[Text Analysis within a Full-Text Search Engine].
```

Comment:
Just a follow-up doubt - Is this (embedding blog links) a recommended practice for the documentation? 
tagging @RayOffiah and @jon-strabala.

Reply:
Ya, will want some advise on this. I embedded this link here per Keshav's recommendation.

Reply:
Eventually we should clean this up, but for now leave it as is.